qcodes/instrument_drivers/stanford_research/SR830.py

```diff
@@ -3,13 +3,73 @@
 from typing import Any
 
 from qcodes import VisaInstrument
-from qcodes.instrument.parameter import ArrayParameter, ParamRawDataType
-from qcodes.utils.validators import Numbers, Ints, Enum, Strings
+from qcodes.instrument.parameter import ArrayParameter, ParamRawDataType, ParameterWithSetpoints, Parameter
+from qcodes.utils.validators import Numbers, Ints, Enum, Strings, Arrays
 
 from typing import Tuple
 import time
 
 
+class ChannelTrace(ParameterWithSetpoints):
+    """
+    Parameter class for the two channel buffers
+    """
+
+    def __init__(self, name: str, channel: int, **kwargs) -> None:
+        """
+        Args:
+            name: The name of the parameter
+            channel: The relevant channel (1 or 2). The name should
+                should match this.
+        """
+        super().__init__(name, **kwargs)
+
+        self._valid_channels = (1, 2)
+
+        if channel not in self._valid_channels:
+            raise ValueError('Invalid channel specifier. SR830 only has '
+                             'channels 1 and 2.')
+
+        if not isinstance(self.root_instrument, SR830):
+            raise ValueError('Invalid parent instrument. ChannelBuffer '
+                             'can only live on an SR830.')
+
+        self.channel = channel
+        self.update_unit()
+
+    def update_unit(self):
```

Comment:
Should type this as `-> None:` since it does not return any thing

Reply:
Yes of course.
I have 3 mypy erros  I don't understand:
qcodes\instrument_drivers\stanford_research\SR830.py:18:5: error: Function is missing a type annotation for one or more arguments
qcodes\instrument_drivers\stanford_research\SR830.py:754:9: error: "_BaseParameter" has no attribute "update_unit"
qcodes\instrument_drivers\stanford_research\SR830.py:848:5: error: Function is missing a type annotation for one or more arguments